rules/layers-slices/slices.test.js

```diff
@@ -12,6 +12,18 @@ const eslint = new ESLint({
 
 describe("Import boundaries between slices and layers", () => {
 
+    describe("IDDQD boundaries", () => {
+        it("should lint without errors in GodMode", async () => {
+            const report = await eslint.lintText(`
+            import { System } from "pages/_system";
+            import { Routes } from "pages/_route";
+            `,
+                {filePath: "src/pages/ui/index.js"});
+
```

Comment:
[`issue:`](https://conventionalcomments.org/) Лучше бы все же более "каноничные" примеры описывать в тесткейсах, а не выдуманные)
(тем более в референсной ишью такое есть)

[`suggestion:`](https://conventionalcomments.org/) Кажется, так гораздо нагляднее и "стабильнее" будет
```suggestion
        it("should lint without errors in GodMode for pages", async () => {
            const report = await eslint.lintText(`
            import { FooPage } from "pages/foo";
            import { BarPage } from "../bar";
            `,
                {filePath: "src/pages/_router/private.routes.js"});

```

[`thoughts:`](https://conventionalcomments.org/) Как видишь, в указанном примере еще добавил относительные импорты - т.к. их тоже тут не стоит забывать 🤔

Reply:
[`suggestion:`](https://conventionalcomments.org/) Плюс, добавил бы еще один тесткейс для компьютедов на всякий (т.к. тесты это и про спеку "эталонности" в том числе)

```js
        it("should lint without errors in GodMode for entities", async () => {
            const report = await eslint.lintText(`
            import { userModel } from "entities/user";
            import { postModel } from "../../Post";
            `,
                {filePath: "src/entities/_computed/UserPost/model.js"});
```

[`thoughts:`](https://conventionalcomments.org/) Тут нужен комментарий, что концепция компьютедов сейчас активно обсуждается кортимой и по сути является экспериментальным подходом по развязыванию сущностей)

> См. подробнее тут
> - https://t.me/feature_sliced/11913
> - https://t.me/feature_sliced/10755